DiamondLightSource/dodal
# Add xspress3mini arming

Fixes #93

To test:
Check and run new unit test
Confirm that the logic of arming the xspress3mini matches the function in gda: armXspress3MiniFluo_detector in mx-config/scripts/zebraBox
